pandas/tests/reshape/test_concat.py

```diff
@@ -1294,6 +1295,16 @@ def test_concat_rename_index(self):
         tm.assert_frame_equal(result, exp)
         assert result.index.names == exp.index.names
 
+    def test_concat_with_ordered_dict(self):
+        # GH 21510
+        ps = pd.concat(OrderedDict([('First', pd.Series(range(3))),
+                                    ('Another', pd.Series(range(4)))]))
+        ps_list = list(ps.keys())
+        exp_list = [('First', 0), ('First', 1), ('First', 2),
+                    ('Another', 0), ('Another', 1),
+                    ('Another', 2), ('Another', 3)]
+        assert ps_list == exp_list
```

Comment:
Typically with test cases we have a pattern of using `result` and `expected` as the variable names. In this case, you should create an `expected` variable that is exactly the Series you are looking for and then us `tm.assert_series_equal(result, expected)` to ensure the proper outcome

Reply:
It is ok if I just change the variables names (expected and result), or I should also create the Series object?

Reply:
Create the Series

Reply:
I'm trying to figure it out how I can create the Series with different sizes without using the concat by itself.

Reply:
I tried to create like this:
```
a = pd.Series(range(3),range(3))
b = pd.Series(range(4),range(4))
expected = pd.Series([a,b],index=['First','Another'])```

Reply:
Done.